### DISCUSSION TITLE
Setting Category for a patch resets the search to page 1

### DISCUSSION THREAD

**Discussion by troach242** [christofmuc/KnobKraft-orm#349] [General]
Now that I have a couple of synthesizers in the Librarian and am really liking it.. I found that if I scroll through patches where the results list is greater then the first page, once I set the category, the list resets back to page 1. Is there a way to get the results list to stay on the selected page when editing the name or category?

**Comment by christofmuc** [comment]
That sounds very much like a regression bug, I am very sure this was not the intention! I will open a bug ticket for this, thanks for reporting!

**Reply by christofmuc** [reply]
Release 2.4.3, which is incoming, contains the fix for this bug. Thanks again for reporting!

**Comment by christofmuc** [comment]
Closing this as fixed.
